src/encryptionService.ts

```diff
@@ -1,119 +1,113 @@
-import { CopilotSettings } from "@/settings/SettingsPage";
 import { Platform } from "obsidian";
 
-// Dynamically import electron to access safeStorage
 // @ts-ignore
-let safeStorage: Electron.SafeStorage | null = null;
+let safeStorageInternal: Electron.SafeStorage | null = null;
 
-if (Platform.isDesktop) {
+function getSafeStorage() {
+  if (Platform.isDesktop && safeStorageInternal) {
+    return safeStorageInternal;
+  }
+  // Dynamically import electron to access safeStorage
   // eslint-disable-next-line @typescript-eslint/no-var-requires
-  safeStorage = require("electron")?.remote?.safeStorage;
+  safeStorageInternal = require("electron")?.remote?.safeStorage;
+  return safeStorageInternal;
 }
 
-export default class EncryptionService {
```

Comment:
we don't need a class if all the classes are static and there is no class state.